src/peft/tuners/adaption_prompt/config.py

```diff
@@ -55,6 +55,13 @@ def is_adaption_prompt(self) -> bool:
         v_proj_layer="v_proj",
         o_proj_layer="o_proj",
     ),
+    "mistral": ModelTypeConfig(  # same for Mistral,
```

Comment:
Regarding the comment: Do you mean "same as llama"?

Reply:
Yes I mean same as llama. 
I can change the comment.